Packs/PaloAltoNetworksCortex_v2/Integrations/PaloAltoNetworksCortex_v2/PaloAltoNetworksCortex_v2.py

```diff
@@ -0,0 +1,786 @@
+    if 'all' not in fetch_subtype:
+        sub_types = [f'sub_type.value = "{sub_type}"' for sub_type in fetch_subtype]
+        query += f' AND ({" OR ".join(sub_types)})'
+    if 'all' not in fetch_severity:
+        severities = [f'vendor_severity.value = "{severity}"' for severity in fetch_severity]
+        query += f' AND ({" OR ".join(severities)})'
+    query += ' LIMIT 100'
+    return query
+
+
+def results_screener(table_name, full_results):
+    """
+    This function is used to make sure we include only pre-defined metrics in the human readable
+    """
+    screened_results = []
+
+    if table_name == "traffic":
+        for row in full_results:
+            screened_row = {metric: value for metric, value in row.items() if metric in TRAFFIC_TABLE_HEADERS}
+            screened_results.append(screened_row)
+    elif table_name == "threat":
+        for row in full_results:
+            screened_row = {metric: value for metric, value in row.items() if metric in THREAT_TABLE_HEADERS}
+            screened_results.append(screened_row)
+    elif table_name == "common":
+        for row in full_results:
+            screened_row = {metric: value for metric, value in row.items() if metric in COMMON_HEADERS}
+            screened_results.append(screened_row)
+    else:
+        return full_results
+
+    return screened_results
+
+
+def get_start_time(date_type, time_value):
+    """
+    Returns datetime of the specified delta from now.
+    For example: if date_type is 'weeks' and time_value is 4- detetime will be 4 weeks ago
+    Args:
+        date_type: Could be either minutes, days or weeks
+        time_value: A number stating how far back the datetime should go
+
+    Returns:
+
+    """
+    current_date = datetime.now().replace(microsecond=0)
+    return current_date - timedelta(**{date_type: time_value})
+
+
+def convert_log_to_incident(log):
+    time_generated = log.get('time_generated')
+    occurred = human_readable_time_from_epoch_time(time_generated)
+    event_id = log.get('id', '')
+    incident = {
+        'name': 'Cortex Event ' + event_id,
+        'rawJSON': json.dumps(log, ensure_ascii=False),
+        'occurred': occurred
+    }
+    return incident
+
+
+''' COMMANDS FUNCTIONS '''
+
+
+def test_module():
+    client = Client()
+    query = 'SELECT * FROM `firewall.traffic` limit 1'
+    client.query_loggings(query)
+    return_outputs('ok')
+
+
+def query_logs_command():
+    """
+    Return the result of querying the Logging service
+    """
+    client = Client()
+    args = demisto.args()
+    query = args.get('query')
+
+    if 'limit' not in query.lower():
+        query += ' LIMIT 100'
+
+    records = client.query_loggings(query)
+
+    table_name = get_table_name(records)
+    screened_results = results_screener('common', records)
+    human_readable = tableToMarkdown('Logs ' + table_name + ' table', screened_results)
+    ec = {
+        'Cortex.Logging(val.id === obj.id)': screened_results
```

Comment:
i see the transformer return `ID` so i think this should be `(val.ID === obj.ID)`